engine/src/conversion/analysis/fun/mod.rs

```diff
@@ -414,116 +448,145 @@ impl<'a> FnAnalyzer<'a> {
         }
     }
 
+    fn add_make_uniques(&mut self, apis: &mut Vec<Api<FnPrePhase>>) {
+        let mut results = Vec::new();
+        'outer: for api in apis.iter() {
+            if let Api::Function {
+                name,
+                fun,
+                analysis,
+                ..
+            } = api
+            {
+                let initial_name = name.clone();
+                if let FnKind::Method(
```

Comment:
I think you can merge the inner `if let` into a single pattern match on the outer `if let`.

Reply:
Done